modules/apps/site/site-cms-site-initializer/src/main/resources/META-INF/resources/js/index.tsx

```diff
@@ -0,0 +1,8 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+// @ts-ignore
```

Comment:
Why do we need the `// @ts-ignore` here?

Reply:
Probably it was a copy-paste, but I'll remove it becuase I'm going to change extension to `.ts`.